.github/workflows/wheel_tests_and_release.yml

```diff
@@ -101,12 +92,20 @@ jobs:
           path: ./dist/*.whl
 
   build_macos_wheels:
-    name: Build wheels on ${{ matrix.os }}
+    name: Build python ${{ matrix.cibw_python }} ${{ matrix.cibw_arch }} wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         os: [macos-latest]
+        cibw_python: [ "cp38-*", "cp39-*", "cp310-*" , "cp311-*" ]
+        cibw_arch: [ "x86_64", "arm64"]
+        include:
+          # only build x86_64 for Python 3.7
+          - os: macos-latest
+            cibw_python: "cp37-*"
+            cibw_manylinux: manylinux2010
```

Comment:
This looks a little off - shouldn't need `manylinux` inside a `macos-latest` block?

Reply:
Yeah, that was a copy-paste error. Actually, I can just go ahead and Python 3.7 now. I checked that we already did so on main in scikit-image.